oneview/resource_server_profile_template.go

```diff
@@ -2707,16 +2822,17 @@ func resourceServerProfileTemplateUpdate(d *schema.ResourceData, meta interface{
 					IsolatedTrunk: rawNetworkItem["isolated_trunk"].(bool),
 					LagName:       rawNetworkItem["lag_name"].(string),
 					Managed:       rawNetworkItem["managed"].(bool),
-					NetworkName:   rawNetworkItem["network_name"].(string),
-					Ipv4:          &ipv4,
+					//					NetworkName:   rawNetworkItem["network_name"].(string),
+					Ipv4: &ipv4,
 				}
 				// checks if all elements are empty
 				// skips connection if empty
-				if !isStructNil(network) {
-					continue
-				}
+				/*				if !isStructNil(network) {
```

Comment:
is this not required now? I mean as it looks like it handle empty connections scenario. It can happen in OV. Not sure how this is handled now?

Reply:
will need to take care of this in next PR